e2e/test/scenarios/question/question-management.cy.spec.js

```diff
@@ -154,13 +154,10 @@ describe(
 
                   moveQuestionTo(/Personal Collection/);
                   assertOnRequest("updateQuestion");
-                  // eslint-disable-next-line no-unscoped-text-selectors -- deprecated usage
-                  cy.contains("37.65");
-
-                  // eslint-disable-next-line no-unscoped-text-selectors -- deprecated usage
-                  cy.contains(
+                  cy.findAllByRole("status").contains(
                     `Model moved to ${getPersonalCollectionName(USERS[user])}`,
                   );
+                  cy.findAllByRole("gridcell").contains("37.65");
```

Comment:
Is it necessary to swap assertion positions to fix the flake? We assert the number before the text previously.

Reply:
Yes. We need to assert the status (i.e. notification) message as soon as `updateQuestion` request ends, because the notification will disappear after a timeout.